GitUI/UserControls/RevisionGrid.Designer.cs

```diff
@@ -161,7 +161,6 @@ private void InitializeComponent()
             // 
             this.Revisions.AllowUserToAddRows = false;
             this.Revisions.AllowUserToDeleteRows = false;
-            dataGridViewCellStyle1.Font = new System.Drawing.Font("Microsoft Sans Serif", 7.8F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(204)));
```

Comment:
The original font is smaller than the system default, and personally I like it more as it is a bit more compact:
![image](https://user-images.githubusercontent.com/4403806/36630171-1a854d18-19b6-11e8-8e16-476e472de6aa.png)

Can we perhaps create a font which is slightly smaller than the system default?

Reply:
Personally I'd rather not go any smaller than 9pt Segoe UI unless the user explicitly chooses it, but I think we can tweak the margins in the revision grid rows so that information density isn't compromised by this change. I'll work on that.

Reply:
Hey David, keen on getting your changes in. Waiting for your tweaks, when you have time.
Thanks

Reply:
Apologies for the delay - other work intervened :) Will push my changes soon.

Reply:
No worries, totally understandable.